extensions/script/ScriptEngine.h

```diff
@@ -41,7 +43,14 @@ class ScriptEngine {
    */
   virtual void evalFile(const std::string &file_name) = 0;
 
+  void setModuleDirectories(std::vector<std::string>&& module_dirs) {
+    module_directories_ = std::move(module_dirs);
+  }
+
   virtual ~ScriptEngine() = default;
+
+ protected:
+  std::vector<std::string> module_directories_;
```

Comment:
This should probably be a vector of `std::filesystem::path` for stronger typing.

Reply:
I'm all for strong types, but I'm not sure if it's worth here as all these paths are used only in string concatenation so we would only convert these to paths just to convert them back to strings when used. Do you see any greater benefits for using path types?

Reply:
If it's that simple, we can leave it as is.